src/pages/RepoPage/ComponentsTab/Header/BranchSelector/BranchSelector.tsx

```diff
@@ -97,8 +105,24 @@ const BranchSelector: React.FC = () => {
             }
           }}
           onSearch={(term: string) => setBranchSearchTerm(term)}
+          disabled={isDisabled}
         />
       </span>
```

Comment:
Add the "latest commit" section under the branch selector.

Reply:
nice!